notebooks/Travel-Gen-AI-Pretrained/03.3_Content_Moderation_Bedrock_Data_Automation.ipynb

```diff
@@ -0,0 +1,402 @@
+    "\n",
+    "Now lets retrieve BDA job results in order to retrieve S3 location with image moderation results.\n",
+    "- If the job is still in progress, it returns the current state (e.g., \"InProgress\"). \n",
+    "- If the job is complete, it returns \"Success\" along with the S3 location of the results. \n",
+    "- If there was an error, it returns \"ServiceError\" or \"ClientError\" with error details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5cbd428-c8f9-4551-9e07-74a7ad33e0ef",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Retrieve BDA job results\n",
+    "\n",
+    "import time\n",
+    "print(\"Wait until successful response with S3 uri location of the job results\")\n",
+    "while True:\n",
+    "    response = bda_runtime.get_data_automation_status(\n",
+    "        invocationArn=invocation_arn\n",
+    "    )\n",
+    "    status = response['status']\n",
+    "    if status not in ['Created', 'InProgress']:\n",
+    "        print(f\" {status}\")\n",
+    "        break \n",
+    "    else:\n",
+    "        print(\".\", end='', flush=True)\n",
+    "        time.sleep(15)\n",
+    "print(json.dumps(response, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4b9ecf4-f506-4c99-aa0a-4ca3733f5288",
+   "metadata": {},
+   "source": [
+    "Lastly, retrieve BDA image processing results. Review `summary` and `content_moderation` nodes and confirm that content moderation capability in BDA detected that the image contains harmful content encouraging smoking."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77cc0e31-4e21-464e-86e0-1c583b17d2a5",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Retrieve BDA image processing results which include summary and content_moderation node\n",
+    "\n",
+    "job_metadata_s3_uri = response['outputConfiguration']['s3Uri']\n",
+    "job_metadata = get_json_object_from_s3_uri(job_metadata_s3_uri)\n",
+    "\n",
+    "for segment in job_metadata['output_metadata']:\n",
+    "    job_s3_uri = segment['segment_metadata'][0]['standard_output_path']\n",
+    "    job_output = get_json_object_from_s3_uri(job_s3_uri)\n",
+    "    print(json.dumps(job_output['image'], indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f40047c4-13d8-4ba2-bf98-c4de5011393b",
+   "metadata": {},
+   "source": [
+    "## Cleanup\n",
+    "\n",
+    "Once experimentation with content moderation using capabilities of Amazon Bedrock Data Automation is completed, run below script to delete demo project in Amazon Bedrock Data Automation"
```

Comment:
Should the cleanup content from the 3.1 lab be replicated here too to remind the user to run through the Cleanup notebook?